server/src/domain/sql_group_backend_handler.rs

```diff
@@ -1,13 +1,16 @@
-use crate::domain::{
-    error::{DomainError, Result},
-    handler::{GroupBackendHandler, GroupListerBackendHandler, GroupRequestFilter},
-    model::{self, GroupColumn, MembershipColumn},
-    sql_backend_handler::SqlBackendHandler,
+use crate::{
+    domain::{
```

Comment:
Oh, something that I didn't realize: we have lldap::domain and lldap_domain?
That's not ideal :/

Reply:
Yeah, didn't really realize it until after lldap_domain.

For one, I think it's worth considering renaming lldap_domain to lldap_domain_types, that's pretty much what it is anyway. It's all types and conversions, if I remember correctly.

Secondly, I think it's worth considering renaming lldap::domain, it's seems a bit like a bucket, currently. I think lldap::model::database, lldap::handlers, and lldap::handlers::ldap, or something like that, might cover most of it, aside from a few stragglers that should probably go to lldap_domain.

Reply:
The architectural vocabulary here would be a shared kernel.

But yeah, I guess lldap_domain_types would be a better name.

And definitely agree on splitting lldap::domain into more refined modules. Potentially even crates! ;)

Reply:
Let's do this in a followup on the "other side" :-)